sdk/src/azure/core/az_aad.c

```diff
@@ -170,13 +170,12 @@ AZ_NODISCARD az_result _az_aad_request_token(_az_http_request* request, _az_toke
   az_span body = { 0 };
   AZ_RETURN_IF_FAILED(az_http_response_get_body(&response, &body));
 
-  uint64_t expires_in_seconds = 0;
+  int64_t expires_in_seconds = 0;
```

Comment:
@antkmsft - we no longer need the unnecessary use of unsigned int64 (and later casting), since we have signed int64 support now.